### PR TITLE
Linear delay

### PR DESCRIPTION
**Pull request by arnopluk**
Changed to delay slider to linear slider (increased size)
Fixed working of Nud for setting delay
Fixed working of hotkeys for increase and decrease of delay
(tested on Windows 10 x64 with IP camera)

### CODE REVIEW

**Comment by JoanCharmant** [comment]
The last commit looks good but I don't think it works 100%. When updating via the NUD a first time it works but it leaves the flag to true, and when updating again from the NUD it doesn't update the slider, doesn't raise the event back to the presenter and doesn't update the internal values.

It's a bit tricky actually. There are other places like this in the codebase where I also have a flag (usually called `manualUpdate`) but rarely with so many sources of updates. In the end I think we can tunnel everything through the slider event:

1. When the update is coming from above in UpdateDelay, it should only change the internal values and never raise the event back, so in UpdateDelayLabel we can remove the entire block that updates the NUD.

2. When the update is coming from the NUD in NudDelay_ValueChanged, we should always force the slider value and let it raise the slider event back, so we guarantee that it will ultimately call presenter.View_DelayChanged.

3. When the update is coming from the slider in SldrDelay_ValueChanged we must always call presenter.View_DelayChanged, and we also should update the NUD value, but only if we are not already in the process of updating it manually.

4. When the update is coming from the commands, we should force the slider value and this should in turn force the NUD value and call presenter.View_DelayChanged. This part is already fine.

To summarize, the following should work:

- In NudDelay_ValueChanged, set a flag right before calling sldrDelay.Force and reset the flag right after the call. The flag can be called "delayUpdating" or something like this. Please start variable names with lowercase and avoid the underscore in the middle unless it's a callback function.
- In SldrDelay_ValueChanged, update the nud value but only if the flag is off. The nud value must be recalculated from the new slider value and the framerate. After that, always call presenter.View_DelayChanged.
- In UpdateDelayLabel, never update the nud value. Since the nud replaced the label, we can probably remove this function altogether.

**Comment by arnopluk** [comment]
The changes you suggested seem to work. I don't have time left to test it today, but maybe tomorrow.
